config_utilities/test/tests/config_arrays.cpp

```diff
@@ -47,7 +47,9 @@
 
 namespace config::test {
 
+namespace {
```

Comment:
Just for my understanding, does this hide it from other tests or what is the idea here?